### DISCUSSION TITLE
Select dropdown value with keyboard input

### DISCUSSION THREAD

**Discussion by rbte** [swiftDialog/swiftDialog#399] [Q&A]
Is there a way to select a dropdown value using the keyboard input? I see that it is possible to select the value when the dropdown is open. Since I would like to use SwiftDialog for some workflows, it would be great to select it without opening the dropdown-list.

Just as a demonstration: If I have the following list (Rabbit, Tiger, Mouse, Monkey) and the focus is on the dropdown, then I would like to select the corresponding entry via the keyboard input (r, t, mou, mon) without first opening the dropdown via "space" and then confirming with "enter".

**Comment by BigMacAdmin** [comment]
This is already possible, with a change to macOS input settings.

System Settings > Keyboard and enable this option for Keyboard Navigation.

Then you can use tab, shift+tab, arrows, and space bar to select your value.

![Screenshot 2024-06-17 at 10 34 15 AM](https://github.com/swiftDialog/swiftDialog/assets/106293503/3fca12d9-bd51-4fa6-a9a1-d9603ef0a952)

**Reply by rbte** [reply]
Thank you very much for your answer. I see that my question may not have been specific enough, sorry about that. I already use this option to navigate. It would be great if one could also use the character keys to select the option without using the "spacebar" (to open) or "enter" (to select and close) the dropdown. I made an example in my initial question.
